spring-rabbit/src/test/java/org/springframework/amqp/rabbit/config/QueueParserIntegrationTests.java

```diff
@@ -74,7 +74,7 @@ public void testArgumentsQueue() throws Exception {
 		assertEquals(null, result);
 
 		connectionFactory.destroy();
-
+		brokerIsRunning.getAdmin().deleteQueue("arguments");
```

Comment:
Can't we mark this `arguments` with the `auto-delete="true"`?
OTOH the `QueueParserIntegrationTests-context.xml` has one more `foo` declaration...